### PR TITLE
Add composition instance chart

### PR DESCRIPTION
**Pull request by nelscata**
This chart will render the Childaccount resources that will deploy the childaccount using the crossplane composition

### CODE REVIEW

File: charts/compositioninstances/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+apiVersion: v2
+name: compositioninstances
```
Comment (jfmrm) [inline charts/compositioninstances/Chart.yaml:2]:
```suggestion
name: composition-instance
```

File: charts/compositioninstances/Chart.yaml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+description: A Helm chart for Kubernetes
```
Comment (jfmrm) [inline charts/compositioninstances/Chart.yaml:3]: Add a descriptive description :D

File: charts/compositioninstances/values.yaml [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+      name: 
+      parentId: ou-8jat-sv0412tm
+      region: us-east-1
+    eks:
+      version: "1.23"
+      nodeSize: # small - medium - large
+      minNodeCount: 
+      maxNodeCount: 
+      desiredNodeCount: 
+      diskSize:
```
[the remark this answers is not in the document]
Comment (jfmrm) [reply charts/compositioninstances/values.yaml:21]: Also add some comments describing what each value will represent at the resulting infrastructure

**Review by jfmrm — CHANGES_REQUESTED** [review]

**Comment by jfmrm** [comment]
@nelscata rename the folder also so it keeps the `-` separator naming pattern

File: charts/compositioninstances/Chart.yaml (hunk 1 above)
Comment (nelscata) [reply charts/compositioninstances/Chart.yaml]: done

File: charts/compositioninstances/Chart.yaml (hunk 2 above)
Comment (nelscata) [reply charts/compositioninstances/Chart.yaml]: done

File: charts/compositioninstances/values.yaml (hunk 3 above)
[the remark this answers is not in the document]
Comment (nelscata) [reply charts/compositioninstances/values.yaml]: done

**Pull request merged**